client/state/comments/actions.js

```diff
@@ -130,21 +130,33 @@ export function changeCommentStatus( siteId, postId, commentId, status ) {
 			siteId,
 			postId,
 			commentId,
+<<<<<<< HEAD
 			status
+=======
+>>>>>>> prettify state + data-layer
```

Comment:
@samouri I believe this should be the only rebase conflict introduced by my PR.
I just added `status` there (https://github.com/Automattic/wp-calypso/pull/16082/files#diff-a1e5174c2173e0927694ec3bd26d79d4), nothing major.